### ISSUE TITLE
ssh not working properly help

### ISSUE THREAD

**Issue by webodan** [meefik/linuxdeploy#966]
I've installed debian just fine using this program on a tablet, but when I log into it, and give my password, it gives me access denied! i've changed the password several times to many different ones, rebooted my tablet and restarted the container but it just won't work. I only get this far using putty.

`android@tablet's passwordl: 
Access denied`

how to change the ssh user password? i know there's one in the settings of this app but this part simply won't work when i change it to a different one then restart the container, wtf?

**Comment by Aeres-u99** [comment]
Are you trying to log in as root? you see mate the root accounts password doesnt work instead use a normal user to ssh into machine and then elevate yourself by typing ```sudo -i```

**Comment by webodan** [comment]
No, i try logging in as the normal user (the user `android` as shown by the log i pasted) and I get that error. I'm using the Play Store app.

**Comment by apieum** [comment]
Hi, just an idea
Have you checked your localization settings ? If they're different you may have different characters for the same code.

**Comment by MyLobin** [comment]
what?

在 2018年7月6日，下午10:22，Grégory Salvan <notifications@github.com<mailto:notifications@github.com>> 写道：


Hi, just an idea
Have you checked your localization settings ? If they're different you may have different characters for the same code.

**Comment by apieum** [comment]
I've seen you can set your locale in linux deploy parameters, and I've faced this kind of problem with putty:

https://serverfault.com/questions/587710/sftp-password-wrong-from-remote-but-not-from-local
